pkg/backup/util/util.go

```diff
@@ -326,23 +334,38 @@ func GetBackupDataPath(provider v1alpha1.StorageProvider) (string, string, error
 	return fmt.Sprintf("%s://%s", string(storageType), backupPath), "", nil
 }
 
-func ValidateBackup(backup *v1alpha1.Backup) error {
-	ns := backup.Namespace
-	name := backup.Name
+func validateAccessConfig(config *v1alpha1.TiDBAccessConfig) string {
+	if config == nil {
+		return "missing cluster config in spec of %s/%s"
+	} else {
+		if config.Host == "" {
+			return "missing cluster config in spec of %s/%s"
+		}
 
-	if backup.Spec.From.Host == "" {
-		return fmt.Errorf("missing cluster config in spec of %s/%s", ns, name)
+		if config.SecretName == "" {
+			return "missing tidbSecretName config in spec of %s/%s"
+		}
 	}
+	return ""
+}
 
-	if backup.Spec.From.SecretName == "" {
-		return fmt.Errorf("missing tidbSecretName config in spec of %s/%s", ns, name)
-	}
+func ValidateBackup(backup *v1alpha1.Backup, tikvImage string) error {
```

Comment:
@DanielZhangQD I refactored the validation logic:
1. If the `backup.Spec.BR` is nil, the `TiDBAccessConfig` will be validated;
2. If the `backup.Spec.BR` presents and `!canSkipSetGCLifeTime(tikvImage)`, the `TiDBAccessConfig` will be validated.